src/www/ui_components/server_connection_viz/server_connection_viz.stories.ts

```diff
@@ -0,0 +1,19 @@
+/* tslint:disable */
+import {ServerConnectionState, ServerConnectionViz} from "./server_connection_viz";
+
+export default {
+  title: "Server Connection Vizualization",
+  component: "server-connection-viz",
+  args: {
+    state: ServerConnectionState.INITIAL,
+  },
+  argTypes: {
+    state: {
+      control: "select",
+      options: Object.keys(ServerConnectionState),
+    },
+  },
+};
+
+export const Example = ({state}: ServerConnectionViz) =>
+  `<server-connection-viz state="${state}"></server-connection-viz>`;
```

Comment:
unfortunately these text-style components make the docs tab kind of broken and useless and we can't seem to turn it off :/